packages/pigeon/lib/java_generator.dart

```diff
@@ -243,6 +243,38 @@ void generateJava(JavaOptions options, Root root, StringSink sink) {
       '@SuppressWarnings({"unused", "unchecked", "CodeBlock2Expr", "RedundantSuppression"})');
   indent.write('public class ${options.className!} ');
   indent.scoped('{', '}', () {
+    for (final Enum enu in root.enums) {
+      indent.writeln('');
+      indent.write('public enum ${enu.name} ');
+      indent.scoped('{', '}', () {
+        int index = 0;
+        for (final String member in enu.members) {
+          indent.writeln('$member($index)${index == enu.members.length - 1 ? ';' : ','}');
```

Comment:
@gaaclarke Should the naming conventions of the target language be followed even if it means it is different than the dart defined names? For example, using ALLCAPS for enumerated members in Java.

Reply:
I suggest we just keep the naming the same as what is defined in the pigeon file.  There is benefits either way, the thing that tips the scale for me is it's simpler to implement and maintain.